### PR TITLE
Support SLE12 on admin node

### PR DESCRIPTION
**Pull request by vmoravec**
Tested succesfully by @vuntz. 
Supersedes  https://github.com/SUSE-Cloud/cct/pull/13 .

### CODE REVIEW

**Comment by vuntz** [comment]
feature:admin:os fails with:

```
    Given the admin node is running correct SLES version # features/step_definitions/admin_node/system_steps.rb:1
      expected "NAME=\"SLES\"\nVERSION=\"12\"\nVERSION_ID=\"12\"\nPRETTY_NAME=\"SUSE Linux Enterprise Server 12\"\nID=\"sles\"\nANSI_COLOR=\"0;32\"\nCPE_NAME=\"cpe:/o:suse:sles:12\"\n" to match "VERSION=12"
      Diff:
      @@ -1,2 +1,8 @@
      -VERSION=12
      +NAME="SLES"
      +VERSION="12"
      +VERSION_ID="12"
      +PRETTY_NAME="SUSE Linux Enterprise Server 12"
      +ID="sles"
      +ANSI_COLOR="0;32"
      +CPE_NAME="cpe:/o:suse:sles:12"
       (RSpec::Expectations::ExpectationNotMetError)
      ./features/step_definitions/admin_node/system_steps.rb:5:in `/^the admin node is running correct SLES version$/'
      features/admin_node.feature:15:in `Given the admin node is running correct SLES version'
```

(rest is working)

File: features/step_definitions/admin_node/system_steps.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
+Given(/^the admin node is running correct SLES version$/) do
+  release_file = admin_node.read_file("/etc/os-release")
   admin_config = config["admin_node"]["os"]
-  expect(release_file).to match(os_name)
-  expect(release_file).to match("VERSION = #{admin_config["version"]}")
-  expect(release_file).to match(
-    "PATCHLEVEL = #{admin_config["patchlevel"]}"
-  )
+  expect(release_file).to match(admin_config["name"])
+  expect(release_file).to match("VERSION=#{admin_config["version"]}")
```
Comment (vuntz) [inline features/step_definitions/admin_node/system_steps.rb]: You need to add quotes:

```
expect(release_file).to match("VERSION=\"#{admin_config["version"]}\"")
```

Comment (vmoravec) [inline features/step_definitions/admin_node/system_steps.rb]: Quotes added.

**Comment by vuntz** [comment]
Tested, +1.

But needs to wait for the SLE12 patches for admin server to land before we merge.

**Comment by dirkmueller** [comment]
please keep this open until I got the PR gating to work on this..

**Comment by dirkmueller** [comment]
+1

**Pull request merged**